erpnext/agriculture/doctype/plant/plant.js

```diff
@@ -7,7 +7,36 @@ frappe.ui.form.on('Plant', {
 			'Harvest': () => frappe.model.open_mapped_doc({
 				method: "erpnext.agriculture.doctype.plant.plant.make_harvest",
 				frm: frm
-			})
+			}),
+			'Destroyed Plant Log': () => { frm.trigger("destroy_plant"); }
 		}
-	}
+	},
+	destroy_plant: (frm) => {
+		frappe.prompt([{
+			fieldname: 'destroy_count',
+			label: __('Destroy Count'),
+			fieldtype: 'Int',
+			reqd: 1,
+		},
+		{
+			fieldname: 'reason',
+			label: __('Reason'),
+			fieldtype: 'Data',
+			reqd: 1,
+		}],
+		(data) => {
+			frm.call('destroy_plant', {
+				destroy_count: data.destroy_count,
+				reason: data.reason
+			}).then(r => {
+				frappe.run_serially([
+					() => frm.reload_doc(),
+					() => frappe.set_route('Form', "Destroyed Plant Log", r.message)
```

Comment:
Why are you reloading doc before routing to different page ?

Reply:
so updated value reflected to the current file

Reply:
- If you are routing to a different page, there is no need to update the values on the current page.

Reply:
Without 'frm.reload_doc()

![plant_reload](https://user-images.githubusercontent.com/56826544/95846374-fb5f0380-0d68-11eb-86fa-0c0c77cda8db.gif)